
public class TestException {
	
	
public static void main(String args[]) {
		
		//Employee e=new Employee();
		
		Employee e=Employee.getEmployee1();


	System.out.println(e);
	
	int no=3;
	System.out.println(no);
	
System.out.println(new Employee().toString());


Employee obj = new Employee();
System.out.println(obj.add1(10, 20));
obj.add2(10, 20);



	}


		
		
	}




